### ISSUE TITLE
Legitimacy of Transport for London feed

### ISSUE THREAD

**Issue by stevevance** [transitland/transitland#242]
This feed [was submitted](https://transit.land/dispatcher/changesets/6208). It contains all tram, underground, overground, and Emirates Air Line service in London. We already have London Underground, which comes in a mega Great Britain feed. 

The feed has old calendar dates, and it's stored on a Google domain. 

I think the changeset should be deleted.

**Comment by rmglennon** [comment]
Thanks for looking into this, @stevevance. The backstory is that a taxi company contacted us by email asking why Underground data was incomplete in routing, and I said that it looks like there are a few stations that are shared with the regional rail data we have. We don't have a full set of Underground data. 

So, the group used this tool https://github.com/CommuteStream/tflgtfs to generate a GTFS file and try to submit it. This looks like a one-off run. I will let the user know that we cannot accept this feed in its current state.

@drewda and I chatted about before I saw this issue. We're thinking we should make some criteria about when to accept a scraped feed and/or turn it into a job that Transitland can do. For example, if it is an established group running the scrape regularly.
